system/Commands/Utilities/Optimize.php

```diff
@@ -52,15 +52,31 @@ final class Optimize extends BaseCommand
      *
      * @var string
      */
-    protected $usage = 'optimize';
+    protected $usage = 'optimize [-c] [-l] [-r]';
+
+    /**
+     * The Command's options
+     *
+     * @var array
+     */
+    protected $options = [
+        'c' => 'Enable config cache',
+        'l' => 'Enable locator cache',
+        'r' => 'Restore default cache settings',
```

Comment:
If you prefer short options, I'm fine with:
```suggestion
        'c' => 'Enable config caching.',
        'l' => 'Enable locator caching.',
        'd' => 'Disable config and locator caching.',
```